lib/shipit.rb

```diff
@@ -103,8 +103,9 @@ def host
     secrets.host.presence
   end
 
-  def github_team
-    @github_team ||= github_oauth_credentials['team'] && Team.find_or_create_by_handle(github_oauth_credentials['team'])
+  def github_teams
+    teams = (Array(github_oauth_credentials['team']) + Array(github_oauth_credentials['teams'])).sort.uniq
+    @github_teams ||= teams.any? && teams.map { |t| Team.find_or_create_by_handle(t) }
```

Comment:
Yes: if there's no team, either we return false here or check for `github_teams` having returned an empty Array in `ShipitController#force_github_authentication`. I preferred the current code.